website/source/docs/auth/aws.html.md

```diff
@@ -33,6 +33,8 @@ for your use cases.
 
 ### EC2 Authentication Method
 
+[![Vault AWS Ec2 Authentication Flow](/assets/images/vault-aws-ec2-auth-flow.png)](/assets/images/vault-aws-ec2-auth-flow.png)
```

Comment:
s/Ec2/EC2/

Again, I realize I'm being a bit nitpicky, but I find it a bit distracting, and the rest of the document capitalizes the `C`.